<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>打分工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="loadDataButton">
         <property name="text">
          <string>加载数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scoreButton">
         <property name="text">
          <string>开始打分</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exportButton">
         <property name="text">
          <string>导出结果</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>得分设置一宫 禄</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_1"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>权</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>科</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>忌</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>存</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>羊</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_6"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>得分设置二宫 禄</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_7"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>权</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_8"/>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>科</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_9"/>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>忌</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_10"/>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>存</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_11"/>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>羊</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_12"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="orderCheckBox">
         <property name="text">
          <string>升序</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="getMaxButton">
         <property name="text">
          <string>二宫取高分</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="sumButton">
         <property name="text">
          <string>二宫取和乘系数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sumFactorEdit">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>50</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>打分数据</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dataTypeComboBox"/>
       </item>
       <item>
        <widget class="QDateEdit" name="beginDateEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="minimumSize">
          <size>
           <width>15</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>15</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>到</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="endDateEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="logTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
